### PR TITLE
DOC-270 PR to update dynamo.md

### PR DESCRIPTION
**Pull request by james-macrometa**
We have made edits for clarity and to correct issues in the code samples. Ticket Reference: https://macrometa.atlassian.net/browse/DOC-270

### CODE REVIEW

File: docs/tutorials/dynamo.md [hunk 1]
```diff
@@ -132,14 +103,31 @@ This flag can be set in `_guestDBs` collection for the corresponding geofabric.
...
+     --request-items '{"Music": [{"PutRequest": {"Item": {"Artist": {"S": "TestName1"},"SongTitle": {"S": "The Best1"}}}},{"PutRequest": {"Item": {"Artist": {"S": 
+"TestName2"},"SongTitle": {"S": "The Best2"}}}}]}' \
      --return-consumed-capacity INDEXES \
      --return-item-collection-metrics SIZE \
      --endpoint-url https://api-gdn.paas.macrometa.io/_api/dynamo
 ```
 
 ### Scan
 
+**Retrieve All Items:**
```
Comment (diana-macrometa) [inline docs/tutorials/dynamo.md]: I suggest changing the bold items in this section to h4s and removing the colons.

**Review by diana-macrometa — APPROVED** [review]
Approved with one minor suggestion.

File: docs/tutorials/dynamo.md (hunk 1 above)
Comment (jamesm-macrometa) [reply docs/tutorials/dynamo.md]: Done

**Comment by jamesm-macrometa** [comment]
Looks good, but we should wait for @james-macrometa to verify my additions before we merge.

**Review by jamesm-macrometa — APPROVED** [review]

**Comment by james-macrometa** [comment]
Hi @jamesm-macrometa and @diana-macrometa I think everything looks good to me. Please proceed.

**Pull request merged**